pybamm/models/base_model.py

```diff
@@ -96,7 +103,7 @@ def __init__(self, name="Unnamed model"):
         # Default behaviour is to use the jacobian and simplify
         self.use_jacobian = True
         self.use_simplify = True
-        self.use_to_python = True
+        self.convert_to_format = "python"
```

Comment:
if casadi performs better should we just use that? though maybe nice to keep the pybamm expression tree by default. maybe it would be good to add a notebook or some kind of readme about using pybamm with casadi to highlight why/when you might want to use this option

Reply:
Will update the install instructions separately in #646 as I've made a few changes there anyway. Casadi is faster than python but harder to debug (not sure how you step through the expression tree to see which bit of code is messing up) so not sure which one should be default? I lean towards casadi

Reply:
Actually, casadi doesn't handle interpolation yet (in pybamm, it has the functionality but needs to be wrapped), and other similar issues might appear, so I will leave python as default until more confident with casadi

Reply:
sounds sensible